sycl/source/detail/pi.cpp

```diff
@@ -53,6 +55,18 @@ namespace pi {
 
 bool XPTIInitDone = false;
 
+void contextSetExtendedDeleter(cl::sycl::context context,
+                               pi_context_extended_deleter func,
+                               void *user_data) {
+  auto impl = getSyclObjImpl(context);
+  auto contextHandle = reinterpret_cast<pi_context>(impl->getHandleRef());
+  auto plugin = impl->getPlugin();
+  plugin.call_nocheck<PiApiKind::piextContextSetExtendedDeleter>(
+      contextHandle, func, user_data);
+  plugin.call_nocheck<PiApiKind::piContextRelease>(
+      contextHandle);
```

Comment:
Please, remove this as you do not call call "get" anymore.

Reply:
DOne